scripts/build_libsnowflakeclient.bat

```diff
@@ -18,6 +18,9 @@ if not "%VisualStudioVersion%"=="14.0" (
     if %ERRORLEVEL% NEQ 0 goto :error
 )
 
+if not exist %scriptdir%\..\deps-build\%arcdir%\aws\ call "%scriptdir%\build_awssdk.bat" %platform% %build_type%
+if %ERRORLEVEL% NEQ 0 goto :error
```

Comment:
Basically, appveyor will build aws sdk every time it runs tests. Since aws sdk on windows is too big such that github rejected the commit.

Reply:
ACK